### ISSUE TITLE
Add documentation about all the upstream objects

### ISSUE THREAD

**Issue by pietroalbini** [pietroalbini/botogram#9]
This is a **very** boring task, so if anyone wants to help... :-)

- [x] Add documentation for ``botogram.User``
- [x] Add documentation for ``botogram.Chat``
- [x] Add documentation for ``botogram.Photo``
- [x] Add documentation for ``botogram.PhotoSize``
- [x] Add documentation for ``botogram.Audio``
- [x] Add documentation for ``botogram.Voice``
- [x] Add documentation for ``botogram.Document``
- [x] Add documentation for ``botogram.Sticker``
- [x] Add documentation for ``botogram.Video``
- [x] Add documentation for ``botogram.Contact``
- [x] Add documentation for ``botogram.Location``
- [x] Add documentation for ``botogram.UserProfilePhotos``
- [x] Add documentation for ``botogram.Message``
- [x] Add documentation for ``botogram.ReplyKeyboardMarkup``
- [x] Add documentation for ``botogram.ReplyKeyboardHide``
- [x] Add documentation for ``botogram.ForceReply``
- [x] Add documentation for ``botogram.Update``

**Comment by tsculpt** [comment]
Hi Pietro, I should be able to help with this...

**Comment by pietroalbini** [comment]
Thank you very much @tsculpt :-)

The documentation about those objects should be at ``docs/api/telegram.rst``, and they don't require digging into the code to document them. Each one (except ``Photo``, which I already documented) is the representation of the object in [the Telegram documentation](https://core.telegram.org/bots/api#available-types). You should maybe rephrase the description of them, add which type they are and if they are optional (as I did in the other objects).

You don't have to do every object, when you do some feel free to send a pull request.

**Comment by pietroalbini** [comment]
Documentation just landed on master. Thanks @tsculpt!
